js/pricing.js

```diff
@@ -28,6 +58,67 @@ $(
                 disabledElement.attr('disabled', true);
                 disabledElement.addClass('disabled');
             }
+        };
+
+        /**
+         * Submit order handler.
+         */
+        const submitOrder = () => {
+            const dataProcessingConsent = $confirmPersonalInformation.prop("checked");
+            const supportAgreementConsent = $confirmDevelopmentAgreement.prop("checked");
+            const data = JSON.stringify({
+                "dataProcessingConsent": dataProcessingConsent,
+                "supportAgreementConsent": supportAgreementConsent
+            });
+
+            const transactionUrl = apiUrl + registerTransactionPath;
+            sendPaymentTransaction (transactionUrl, data);
+        };
+
+        /**
+         * Sends the transaction data and returns the transaction ID. If the request is successful,
+         * redirects to the Payment screen.
+         *
+         * @param {String} transactionUrl - the API URL
+         * @param {Object} data - the transaction data
+         */
+        const sendPaymentTransaction = (transactionUrl, data) => {
+            showRedirect(false);
+            $.ajax(transactionUrl, {
+                type: 'POST',
```

Comment:
Can we use `$.post` instead? (please check the docs: https://api.jquery.com/jquery.post/)

Reply:
Could we discuss it vocally?